
The path_length needs to be initialized to the argv specified path: the printf
predicate does use it to strip this initial path of the filename.

  Rene Rebe <rene@exactcode.de>

--- findutils-4.2.10/find/find.c	2004-12-05 22:59:27.000000000 +0100
+++ findutils-4.2.10-fixed/find/find.c	2004-12-24 17:47:51.473678472 +0100
@@ -1015,6 +1015,8 @@
 static void
 process_top_path (char *pathname)
 {
+  path_length = strlen (pathname);
+  curdepth = 0;
   process_path (pathname, pathname, false, ".");
 }
 
